tests/unit/dashboard-controller.js

```diff
@@ -0,0 +1,69 @@
+describe('LoginController', function() {
+	var $controller,
+		rulesData = [
+			{
+				id: "001",
+				name: "Published and unpublished match",
+				last_updated: "2015-08-18T06:36:40+00:00",
+				last_published: "2015-08-18T06:36:40+00:00"
+			}, {
+				id: "002",
+				name: "Updated, then published",
+				last_updated: "2015-08-18T06:36:40+00:00",
+				last_published: "2015-08-28T16:30:45+00:00"
+			}, {
+				id: "003",
+				name: "Updated more recently than published",
+				last_updated: "2015-08-18T06:36:40+00:00",
+				last_published: "2015-08-08T16:30:45+00:00"
+			}, {
+				id: "004",
+				name: "Never published",
+				last_updated: "2015-08-18T06:36:40+00:00",
+				last_published: null
+			}
+		],
+		vm;
+
+	beforeEach(module('ucApp'));
+
+	beforeEach(inject(function(_$controller_) {
+		$controller = _$controller_;
+	}));
+
+	describe('vm.ruleIsPublished', function() {
```

Comment:
like these tests. I wonder if this functionality doesn't belong on the rules service though?